File: cases/taint/javascript/argument-position-positive/ArgumentPositionPositive.js
function dfb_source() { // DFB-SOURCE: argument-position-input
  return "tainted";
}

function chooseFirst(first, second) { // DFB-WITNESS: argument-position-first
  return first;
}

function dfb_sink(value) {} // DFB-SINK: argument-position-sink

function run() {
  const result = chooseFirst(dfb_source(), "clean");
  dfb_sink(result);
}

File: cases/taint/rust/argument-position-positive/argument_position_positive.rs
fn dfb_source() -> i32 { // DFB-SOURCE: argument-position-input
    1
}

fn choose_first(first: i32, second: i32) -> i32 { // DFB-WITNESS: argument-position-first
    first
}

fn dfb_sink(value: i32) {} // DFB-SINK: argument-position-sink

fn run() {
    let result = choose_first(dfb_source(), 0);
    dfb_sink(result);
}
